.github/workflows/perf.yml

```diff
@@ -161,3 +171,4 @@ jobs:
           destination_branch: "main"
           github_token: ${{ secrets.GITHUB_TOKEN }}
           pr_title: "Update Performance Model"
+          pr_body: "Generated by performance test workflow [#${{ github.run_number }}](https://github.com/aws-observability/aws-otel-collector/actions/runs/${{ github.run_id }}) using https://github.com/aws-observability/aws-otel-collector/commit/${{ needs.get-testing-version.outputs.commit_id }}."
```

Comment:
Commit id is the sha?

Reply:
Ignore this comment

Reply:
Yep, so an example PR body would be:

Generated by performance test workflow [#69](https://github.com/aws-observability/aws-otel-collector/actions/runs/1404323726) using https://github.com/aws-observability/aws-otel-collector/commit/cc8eb315135bdfed89782ee2f1368e5445df5ed3.